playground/src/config/workspaces/policy.ts

```diff
@@ -3,7 +3,7 @@ import { OPTION_TYPE, optionCUE, optionJSON, optionTerminal, optionYAML } from '
 import { deepFreeze } from '@helpers/deep-freeze';
 
 export const policyWorkspace: Workspace = deepFreeze<Workspace>({
-    enabled: false,
+    enabled: true,
```

Comment:
Don't forget to back this out before we import the PR.

Reply:
oops, accidentally committed that...   I put it back to false